_posts/en/newsletters/2020-03-04-newsletter.md

```diff
@@ -0,0 +1,226 @@
+  new peer.  The new message would make it easier for a node to
+  negotiate what features it wants to receive from its peer.  A
+  challenge here is that previous versions of Bitcoin Core would
+  terminate a new connection if certain messages didn't appear in a
+  particular order---and it's into this strict sequence that Daftuar
+  wants to insert a new message.  The proposal does increment the P2P
+  protocol version to provide backwards compatibility, but Daftuar is
+  seeking feedback from maintainers of full nodes about whether the
+  insertion of negotiation messages would cause any problems.  If you're
+  aware of any problems, please reply to the thread.
+
+- **Proposal to standardize an anti covert nonce protocol:** Stepan
+  Snigirev [started][snigirev nonce] a discussion on the Bitcoin-Dev
+  mailing list about standardizing a protocol that prevents hardware
+  wallets from generating a signature that uses a biased nonce to leak
+  the user's private key.  One [previously proposed][sign to contract]
+  mechanism for defending against this attack would be to use the *sign
+  to contract protocol* to verify that the signature commits to
+  randomness selected by the hardware wallet's host computer or mobile
+  device.  Developers of libsecp256k1 have already been working on an
+  API to both [enable generic sign to contract][secp s2c] and build an
+  [anti covert nonce capability][secp nonce] on top of it.  Snigirev's
+  email describes the current preferred protocol and how it can be
+  extended to multiple computers and Partially Signed Bitcoin
+  Transactions ([PSBTs][topic psbt]).
+
+- **Taproot in the generic group model:** Lloyd Fournier published his
+  [poster][fournier poster] for the Financial Cryptography conference
+  two weeks ago describing what properties must be present in the hash
+  function used with taproot in order for taproot to be secure.  This
+  extends a previous [proof][poelstra proof] by Andrew Poelstra which
+  made the broader assumption that the hash function acted as a [random
+  oracle][].  Those evaluating the cryptographic security of taproot are
+  encouraged to review Poelstra's proof and Fournier's poster.
+
+## Highlights from the Bitcoin Core PR Review Club
+
+The [Bitcoin Core PR Review Club][] is a weekly IRC meeting for newer
+contributors to the Bitcoin Core project to learn about the review process. An
+experienced Bitcoin Core contributor provides background information about a
+selected PR and then leads a discussion on IRC.
+
+The Review Club is an excellent way to learn about the Bitcoin protocol, the
+Bitcoin Core reference implementation, and the process of making changes to
+Bitcoin. Notes, questions and meeting logs are posted on the website for those
+who are unable to attend in real time, and as a permanent resource for those
+wishing to learn about the Bitcoin Core development process.
+
+In this section, we summarise some recent Review Club meetings.
```

Comment:
I'm sympathetic to the British "ise" but find it a bit strange to mix "ize" and "ise" in the same newsletter without mentioning different authors for each part, otherwise people may think one of the two styles is a typo :)

Reply:
Yes, the spelling should be consistent within a document, and Dave is the main author of this newsletter, so I should follow his preference.